docs/Available-Symbols-Generators.md

```diff
@@ -1,15 +1,26 @@
-Inside the `template.jon` file, you can define custom symbols that will be used inside the template files. Some of these symbols define parameters that you can input from command line, others are result of logical expressions applied to symbols, and others are generated using **Macros**.     
-To use a Macro inside your `template.json` you have to add the `"type": "generated"` to the symbol definition, and use the `"generator": ...` parameter to select the macro to use.    
-This is a sample of definition of a symbol with a macro, the `port` Macro, that generates a random number for an http port.    
+Inside the `template.json` file, you can define custom symbols that will be used inside the template files. 
+The supported symbol types are:
+- Parameter - a symbol which value is typically provided by the user when creating the template. If the value is not provided, then the value is taken from host configuration, otherwise default value is used. 
```

Comment:
If not provided the value is taken from the host configuration, default value used otherwise.